fastlane/spec/actions_specs/import_from_git_spec.rb

```diff
@@ -305,6 +305,29 @@ def self.is_supported?(platform)
         end
       end
 
+      it "works with one HTTP header" do
+        header = 'Authorization: Basic my_base_64_key'
+
+        allow(Fastlane::Actions).to receive(:sh).and_call_original
+        expect(Fastlane::Actions).to receive(:sh).with(a_string_including("--config \"http.extraHeader=#{header}\""))
```

Comment:
If you make the change I suggest above to call `Actions.sh` with an `Array` to add security and prevent the risk of shell injection and mis-escaping, I think you can use `a_collection_including(['--config', "http.extraHeader=#{header}"])` here instead of `a_string_including` to make the spec example pass.